runtime/compiler/p/codegen/J9CodeGenerator.cpp

```diff
@@ -353,6 +353,12 @@ bool J9::Power::CodeGenerator::suppressInliningOfRecognizedMethod(TR::Recognized
           }
       }
 
+   if (method == TR::java_util_zip_CRC32C_updateBytes ||
+       method == TR::java_util_zip_CRC32C_updateDirectByteBuffer)
+      {
+      return true;
+      }
+
```

Comment:
Can we add a comment here that says something like "unlike CRC32's update* methods which are JNI natives, CRC32C's update* methods are written in Java and therefore we need to prevent the inliner from eliminating the calls so we can redirect them to our optimized helpers".

I had to look around a bit to understand why we weren't doing this for CRC32 so comments will be helpful to future readers.